plugins/templates/templates/default.js

```diff
@@ -14,74 +14,41 @@ CKEDITOR.addTemplates( 'default', {
 		title: 'Image and Title',
 		image: 'template1.gif',
 		description: 'One main image with a title and text that surround the image.',
-		html: '<h3>' +
-			// Use src=" " so image is not filtered out by the editor as incorrect (src is required).
-			'<img src=" " alt="" style="margin-right: 10px" height="100" width="100" align="left" />' +
-			'Type the title here' +
-			'</h3>' +
-			'<p>' +
-			'Type the text here' +
-			'</p>'
+		html: getHtml('image-and-title.html')
 	},
 	{
 		title: 'Strange Template',
 		image: 'template2.gif',
 		description: 'A template that defines two columns, each one with a title, and some text.',
-		html: '<table cellspacing="0" cellpadding="0" style="width:100%" border="0">' +
-			'<tr>' +
-				'<td style="width:50%">' +
-					'<h3>Title 1</h3>' +
-				'</td>' +
-				'<td></td>' +
-				'<td style="width:50%">' +
-					'<h3>Title 2</h3>' +
-				'</td>' +
-			'</tr>' +
-			'<tr>' +
-				'<td>' +
-					'Text 1' +
-				'</td>' +
-				'<td></td>' +
-				'<td>' +
-					'Text 2' +
-				'</td>' +
-			'</tr>' +
-			'</table>' +
-			'<p>' +
-			'More text goes here.' +
-			'</p>'
+		html: getHtml('strange-template.html')
 	},
 	{
 		title: 'Text and Table',
 		image: 'template3.gif',
 		description: 'A title with some text and a table.',
-		html: '<div style="width: 80%">' +
-			'<h3>' +
-				'Title goes here' +
-			'</h3>' +
-			'<table style="width:150px;float: right" cellspacing="0" cellpadding="0" border="1">' +
-				'<caption style="border:solid 1px black">' +
-					'<strong>Table title</strong>' +
-				'</caption>' +
-				'<tr>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-				'</tr>' +
-				'<tr>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-				'</tr>' +
-				'<tr>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-				'</tr>' +
-			'</table>' +
-			'<p>' +
-				'Type the text here' +
-			'</p>' +
-			'</div>'
-	} ]
-} );
+		html: getHtml('text-and-table.html')
+	}
+	]
+});
+
+// Get the contents of the given html file.
+function getHtml(fileName) {
+	let path = CKEDITOR.getUrl(CKEDITOR.plugins.getPath('templates') + 'templates/html/' + fileName);
+	return fetch(path)
+		.then(response => {
+			if (!response.ok) {
+				throw Error(response.statusText);
+			}
+			return response;
+		})
+		.then(response => response.arrayBuffer())
+		.then(buffer => {
+		    let decoder = new TextDecoder("iso-8859-1");
+		    let text = decoder.decode(buffer);
+		    return text;
+		})
```

Comment:
There is no need to treat data as binary. Additionally it should be always treated as encoded in UTF-8, not ISO-8859-1. It can be accomplished by using [`CKEDITOR.ajax#loadText()` function](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_ajax.html#method-loadText) from `ajax` plugin.